src/Blocks/custom/featured-posts/components/featured-posts-editor.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { LoadMoreEditor as LoadMoreEditorComponent } from '../../../components/load-more/components/load-more-editor';
+
+export const FeaturedPostsEditor = ({ attributes, setAttributes }) => {
+	return (
+		<LoadMoreEditorComponent
```

Comment:
why is FeaturedPostsEditor loading only LoadMoreEditorComponent?

Where is the part for the FeaturedPostsEditor?

Reply:
It's only `LoadMoreEditorComponent` because I didn't have anything extra to add for Featured Posts, but I still wanted the editable button text from the Load More editor